pages/_type/_id/edit.vue

```diff
@@ -382,55 +379,103 @@
+              <a
+                href="https://spdx.org/licenses/"
+                target="_blank"
+                rel="noopener noreferrer"
+                class="text-link"
+                >SPDX license identifier</a
+              >
+              in the marked area. If your license does not have a SPDX
+              identifier (for example, if you created the license yourself or if
+              the license is Minecraft-specific), simply check the box and enter
+              the name of the license instead.
+              <br />
+            </span>
+            Confused? See our
+            <a
+              href="https://blog.modrinth.com/licensing-guide/"
+              target="_blank"
+              rel="noopener noreferrer"
+              class="text-link"
+            >
+              licensing guide</a
+            >
+            for more information.
+          </span>
+        </label>
         <div class="legacy-input-group">
           <Multiselect
+            id="license-multiselect"
             v-model="license"
-            placeholder="Choose license..."
+            placeholder="Select license..."
             track-by="short"
-            label="short"
-            :options="$tag.licenses"
-            :custom-label="(value) => value.short.toUpperCase()"
+            label="friendly"
+            :options="defaultLicenses"
             :searchable="true"
             :close-on-select="true"
             :show-labels="false"
+            :class="{
+              'known-error': license.short === '' && showKnownErrors,
+            }"
             :disabled="
               (currentMember.permissions & EDIT_DETAILS) !== EDIT_DETAILS
             "
           />
+          <Checkbox
+            v-if="license.requiresOnlyOrLater"
+            v-model="allowOrLater"
+            :disabled="
+              (currentMember.permissions & EDIT_DETAILS) !== EDIT_DETAILS
+            "
+          >
+            Allow later editions of this license
+          </Checkbox>
+          <Checkbox
+            v-if="license.friendly === 'Custom'"
+            v-model="nonSpdxLicense"
+            :disabled="
+              (currentMember.permissions & EDIT_DETAILS) !== EDIT_DETAILS
+            "
+          >
+            License does not have a SPDX identifier
+          </Checkbox>
           <input
-            v-model="license_url"
-            type="url"
+            v-if="license.friendly === 'Custom'"
+            v-model="license.short"
+            type="text"
             maxlength="2048"
-            placeholder="License URL"
+            :placeholder="nonSpdxLicense ? 'License name' : 'SPDX identifier'"
             :class="{
-              'known-error': newProject.license_url === '' && showKnownErrors,
+              'known-error': license.short === '' && showKnownErrors,
             }"
             :disabled="
               (currentMember.permissions & EDIT_DETAILS) !== EDIT_DETAILS
             "
           />
```

Comment:
In the UI it is not very intuitive that you don't have to set a license URL anymore